src/ragbuilder/langchain_module/retriever/retriever.py

```diff
@@ -35,7 +35,16 @@ def getRetriever(**kwargs):
 
     if retriever_type in ["vectorSimilarity", "vectorMMR"]:
         logger.info("Vector Retriever Invoked")
-        code_string = f"""retriever=c.as_retriever(search_type='{kwargs['search_type']}', search_kwargs={{'k': {kwargs['search_kwargs']}}})"""
+        document_compressor_pipeline=kwargs['retriever_kwargs'].get('document_compressor_pipeline',None)
+        if document_compressor_pipeline is not None:
+            if 'CrossEncoderReranker' in document_compressor_pipeline:
```

Comment:
Does it make sense to do this for all compressors? Esp. EmbeddingsFilter, LLMChainFilter etc. can all show impact only when there's a good number of candidate documents on which this logic can be applied.
What do you think?